app/src/main/AndroidManifest.xml

```diff
@@ -28,6 +28,9 @@
             android:name=".recipepicker.BlankActivity"
             android:screenOrientation="portrait" />
 
+        <activity
+            android:name=".googleLogin.GoogleLoginActivity"
```

Comment:
Package names don't use `snakeCase`. Instead they use everything in lowercase. In this context, it would be `googlelogin`.

The cool thing is that you can rename the package and have all its references refactored by using the IDE `Rename` functionality. 😉